### PR TITLE
{bio}[GCC/11.3.0] add libSBML 5.19.7 

### PR DESCRIPTION
**Pull request by jkuhl-uni**

### CODE REVIEW

File: easybuild/easyconfigs/l/libSBML/libSBML-5.19.7-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+configure_cmd += '-DWITH_CHECK=ON -DWITH_SWIG=ON -DWITH_PERL=ON -DWITH_PYTHON=OFF '
+configure_cmd += '-DWITH_EXPAT=ON -DWITH_LIBXML=OFF -Wno-dev '
+configure_cmd += '-DENABLE_{LAYOUT,QUAL,COMP,FBC,RENDER,GROUPS,MULTI,DISTRIB}=ON '
+configure_cmd += '../%(namelower)s-%(version)s'
+
+
+runtest = 'check'
+
+sanity_check_paths = {
+    'files': ['lib64/libsbml.so', 'lib64/perl5/site_perl/5.34.1/x86_64-linux-thread-multi/LibSBML.pm'],
```
Comment (casparvl) [inline easybuild/easyconfigs/l/libSBML/libSBML-5.19.7-GCC-11.3.0.eb:66]: please use `lib64/libsbml.%s` % SHLIB_EXT instead of hardcoding `.so`.

Comment (jkuhl-uni) [reply easybuild/easyconfigs/l/libSBML/libSBML-5.19.7-GCC-11.3.0.eb:66]: Hi, sure, I just heard of the new policy in my other pr.
I'll fix it later.

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen3

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/d571a584c526973061d0aa073f43579e for a full test report.

**Comment by casparvl** [comment]
Going in, thanks @jkuhl-uni!

**Comment by jkuhl-uni** [comment]
Thanks for the quick replies and collaboration :)

**Pull request merged**